src/static/client/app/signup/signup.component.scss

```diff
@@ -1,110 +1,129 @@
+    height: auto;
+    display: block;
+    margin: 0 auto 20px auto; // This centers the image and adds space below it.
+  }
+
+  .wrapper {
+    max-width: 400px;
+    background-color: #fff;
+    border-radius: 10px;
+    box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1), 0 1px 3px rgba(0, 0, 0, 0.08);
+    padding: 30px;
+    width: 100%;
+    max-width: 350px;
+
+    .h2 {
+      text-align: center;
+      margin-bottom: 24px;
     }
-  
-    .wrapper {
-      max-width: 400px;
-      background-color: #fff;
-      border-radius: 10px;
-      box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1), 0 1px 3px rgba(0, 0, 0, 0.08);
-      padding: 30px;
-      width: 100%;
-      max-width: 350px;
-  
-      .h2 {
-        text-align: center;
-        margin-bottom: 24px;
-      }
-  
-      #formContent {
-        display: flex;
-        flex-direction: column;
-  
-        .form-label {
-          display: block;
-          font-size: 14px;
-          font-weight: 600;
-          color: #4a4a4a;
+
+    #formContent {
+      display: flex;
+      flex-direction: column;
+
+      .fadeIn.first {
+        margin-top: 30px; // Added to give some space above the logo
+
+        .info {
+          font-size: 10px; // Decrease the font size for the instructions
           margin-bottom: 10px;
         }
-        
-  
-        .form-input {
-          width: 100%;
-          padding: 10px;
-          font-size: 14px;
-          border: 1px solid #e0e0e0;
-          border-radius: 5px;
-          outline: none;
-          transition: border-color 0.3s;
-          margin-bottom: 20px;
-        
-          &:focus {
-            border-color: #007bff;
-          }
-        }
-        .check-lg {
-          margin-bottom: 20px;
-        }
-  
-        .form-button {
-          padding: 10px;
-          font-size: 16px;
-          font-weight: 600;
-          border: none;
-          border-radius: 5px;
-          cursor: pointer;
-          outline: none;
-          transition: background-color 0.3s;
-          margin-bottom: 10px;
-        
-          &:disabled {
-            background-color: #ccc;
-          }
+      }
+
+      .form-label {
+        display: block;
+        text-align: center;
+        font-size: 18px; // Increased from 14px to 18px for bigger labels
+        font-weight: 600;
+        color: #4a4a4a;
+        margin-bottom: 10px;
+      }
+
+
+      .form-input {
+        width: 100%;
+        padding: 10px;
+        font-size: 14px;
+        border: 1px solid #e0e0e0;
+        border-radius: 5px;
+        outline: none;
+        transition: border-color 0.3s;
+        margin-bottom: 20px;
+
+        &:focus {
+          border-color: #007bff;
         }
-  
-        .blue-button {
-          background-color: #1632AA;
-          color: #fff;
-          margin-bottom: 15px;
-  
-          &:hover {
-            background-color: #0056b3;
-          }
+      }
+
+      .check-lg {
+        margin-bottom: 20px;
+      }
```

Comment:
Can you use the bootstrap mb-3 class for this?

Reply:
Sure, using Bootstrap's utility classes like mb-3 for margin-bottom is a great idea. It helps to keep the code clean and we can leverage the inbuilt Bootstrap styles. I'll replace the margin-bottom: 20px; with mb-3 which will give a margin-bottom of 1rem. Thank you for pointing it out.

Reply:
Sure, using Bootstrap's utility classes like mb-3 for margin-bottom is a great idea. It helps to keep the code clean and we can leverage the inbuilt Bootstrap styles. I'll replace the margin-bottom: 20px; with mb-3 which will give a margin-bottom of 1rem. Thank you for pointing it out

Reply:
it doesn't show any changes on this file